bundles/io/org.eclipse.smarthome.io.net.test/src/test/java/org/eclipse/smarthome/io/net/http/internal/WebClientFactoryTest.java

```diff
@@ -66,32 +63,6 @@ public void testGetClients() throws Exception {
         webClientFactory.deactivate();
     }
 
-    @Test
-    public void testGetHttpClientWithEndpoint() throws Exception {
```

Comment:
as long as we keep the method in place (deprecated), we should imho also leave the tests in place.

Reply:
I would like to make it behave as if no endpoint was given, so to keep a test for that behavior in place seems odd to me. Would you prefer to also keep the behavior? 

I’m planning to add new tests for the new method before marking it complete.

Reply:
> Would you prefer to also keep the behavior?

YES! That's what backward compatibility is about :-)